### ISSUE TITLE
Does not discover all devices

### ISSUE THREAD

**Issue by hotab** [MoshiBin/ssdpy#46]
Adding a few newlines here:
```
    data = (
        "M-SEARCH * HTTP/1.1\r\n"
        "HOST:{}\r\n"
        'MAN: "ssdp:discover"\r\n'
        "ST:{}\r\n"
        "MX:{}\r\n"
    ).format(host, st, mx)
```
e.g.:
```
    data = (
        "M-SEARCH * HTTP/1.1\r\n"
        "HOST:{}\r\n"
        'MAN: "ssdp:discover"\r\n'
        "ST:{}\r\n"
        "MX:{}\r\n\r\n"
    ).format(host, st, mx)
```

Leads to discovering more devices (seen at least 1 such device).

**Comment by hotab** [comment]
UDP: this seems fixed in master, but not released to pip. Can we have an update?

**Comment by MoshiBin** [comment]
Released version 0.3.0 which includes #42 that fixes this issue.

**Comment by hotab** [comment]
Thanks!
